antsibull_changelog/cli.py

```diff
@@ -411,7 +411,7 @@ def command_release(args: Any) -> int:
     flatmap = _determine_flatmap(collection_details, config)
 
     if not version or not codename:
-        if not config.is_collection:
+        if not config.is_collection and not paths.is_other_project:
```

Comment:
I'm not sure if this is necessary? Only collections will evaluate to true here, with the changes below which also check `is_collection` should be sufficient.

Reply:
This is necessary since this code path only needs to be run for ansible-core/-base, but neither for collections nor for other projects.